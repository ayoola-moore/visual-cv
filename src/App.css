body {
  display: flex;
  justify-content: center;
}

button {
  padding: 2% 10%;
  background: #13131f;
  color: white;
  border: none;
  border-radius: 4px;
}

button:disabled {
  background: #ccc;
}

.App {
  text-align: center;
  padding: 10%;
  border: 1px solid;
  width: 100%;
  min-width: 350px;
  box-sizing: border-box;
  margin-top: 15%;
}
.input-wrapper {
  padding: 2% 0;
  text-align: left;
}

.input-wrapper > * {
  display: block;
}

.messageDefault {
  border: 1px solid #333;
  min-width: 300px;
  padding: 5%;
  margin: 10% 0;
  border-radius: 2px;
}

.error {
  color: #ed0f0f;
}

.success {
  background: #b4f1b4;
}

input {
  min-width: 300px;
  width: 90%;
  padding: 10px;
}

/* * {
  border: 1px solid red;
} */

form {
  width: 100%;
  padding: 0;
}

/* input {
  width: auto;
} */

#root {
  display: flex;
  flex-wrap: wrap;
  justify-content: center;
  min-height: 100vh;
  align-items: center;
  width: 100%;
}

.input-wrapper {
  display: block;
  padding: 5% 2% 10%;
  width: 50%;
  min-width: 320px;
  max-width: 600px;
  margin: 0 auto;
  border: 1px solid grey;
  border-radius: 4px;
}
