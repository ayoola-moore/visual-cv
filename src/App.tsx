import React from "react";
import "./App.css";
import Form from "./components/Form/index";

function App() {
  return <Form />;
}

export default App;
