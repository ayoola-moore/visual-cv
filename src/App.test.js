import MatchMediaMock from "jest-matchmedia-mock";
import React from "react";
import renderer from "react-test-renderer";
import App from "./App";

let matchMedia;

describe("Your testing module", () => {
  beforeAll(() => {
    matchMedia = new MatchMediaMock();
  });

  afterEach(() => {
    matchMedia.clear();
  });

  it("App renders correctly", () => {
    const tree = renderer.create(<App />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
