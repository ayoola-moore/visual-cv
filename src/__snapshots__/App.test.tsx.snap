// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `
<form
  onSubmit={[Function]}
>
  <div
    className="input-wrapper"
  >
    <label
      htmlFor="firstNumber"
    >
      First Number
    </label>
    <input
      id="firstNumber"
      name="firstNumber"
      onBlur={[Function]}
      onChange={[Function]}
      placeholder="Enter first number"
      type="number"
    />
    <div
      className="error"
    />
    <br />
    <label
      htmlFor="secondNumber"
    >
      Second Number
    </label>
    <input
      id="secondNumber"
      name="secondNumber"
      onBlur={[Function]}
      onChange={[Function]}
      placeholder="Enter second number"
      type="number"
    />
    <div
      className="error"
    />
    <br />
    <button
      disabled={true}
      type="submit"
    >
      Submit
    </button>
  </div>
</form>
`;
